src/parser/jobs/drg/modules/Buffs.tsx

```diff
@@ -112,6 +114,13 @@ export default class Buffs extends Analyser {
 		}
 	}
 
+	private onDs(event: Events['action']) {
```

Comment:
Minor nit, might as well use the full `onDragonSight` name for the method since the shorthand just makes it confusing for reviewers. I can't block on it, but something to consider.

Reply:
makes sense to me, easy fix :)